motor/core.py

```diff
@@ -1065,23 +1065,17 @@ async def watch_collection():
           def main():
               loop = IOLoop.current()
               # Start watching collection for changes.
-              loop.add_callback(watch_collection)
-              try:
-                  loop.start()
-              except KeyboardInterrupt:
-                  pass
-              finally:
-                  if change_stream is not None:
-                      change_stream.close()
+            try:
+                loop.run_sync(watch_collection)
+            except KeyboardInterrupt:
+                if change_stream:
+                   loop.run_sync(change_stream.close)
 
           # asyncio
           try:
-              asyncio.run(watch_collection)
+              asyncio.run(watch_collection())
           except KeyboardInterrupt:
-                  pass
-          finally:
-              if change_stream is not None:
-                  change_stream.close()
+              pass
```

Comment:
Does this pass need to be here?

Reply:
Yes, that or a comment, or it would be a syntax error.